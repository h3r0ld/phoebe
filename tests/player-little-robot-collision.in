reset
hardworking-little-robot-random off
robot-move 0 22 6
move W