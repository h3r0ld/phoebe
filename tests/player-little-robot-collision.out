Hardworking-little-robot 0: New Position: Cell(22, 6)
Player 0: Cell(22,8) Speed: 2 CanChangeDirection: True
Player 0: New Position: Cell(22, 6)
Cell(22, 6): GameObject: OilStain